pengwin-setup.d/common.sh

```diff
@@ -52,6 +52,21 @@ echo "Removing unnecessary packages."
 sudo apt-get autoremove -y
 }
 
+function command_check() {
+# Usage: command_check <EXPECTED PATH> <ARGS (if any)>
```

Comment:
Please indent with 2 spaces

Reply:
should be resolved now, will make another PR once this one has gone through fixing up the rest of the spacing inconsistencies throughout the file with other functions